combat-trainer.lic

```diff
@@ -1171,6 +1181,15 @@ class AbilityProcess
         next unless !timer || (Time.now - timer).to_i > 30
         game_state.cooldown_timers[name] = Time.now if activate_khri?(@kneel_khri, name)
       end
+    @barb_buffs.each do |name|
+      if !(Flags["ap-#{name}-active"] == true)
+        case DRC.bput("#{get_data('spells').barb_abilities[name]['activate']}", "#{get_data('spells').barb_abilities[name]['start']}", 'But you are already', 'Your inner fire lacks')
+        when 'But you are already', "#{get_data('spells').barb_abilities[name]['start']}"
+          Flags["ap-#{name}-active"] = true
+        when 'Your inner fire lacks'
```

Comment:
Left it here to discuss some sort of defense against lacking IF. Do you think it's needed? Should we leave it up to the player to determine their max IF usage? I opt for the latter.